PeriodogramAnalysis/test_spectrogram_analysis.py

```diff
@@ -0,0 +1,73 @@
+from typing import Generator, Any
+
+import pytest
+import numpy as np
+from scipy.signal import spectrogram
+from miv.core.datatype import Signal
+from spectrogram_analysis import SpectrogramAnalysis
+
+
+def signal_input():
```

Comment:
I see some copy and paste here

Reply:
Forgot `mock_signal_generator` can be imported.. now this signal generator is imported here `from test_power_density_statistics import mock_signal_generator`